tests/drivers/clock_control/nrf2_clock_control/prj.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_ZTEST=y
+
+CONFIG_DEBUG_OPTIMIZATIONS=y
+CONFIG_MAIN_STACK_SIZE=8192
+
+CONFIG_CLOCK_CONTROL_NRF2_NRFS_DVFS_TIMEOUT_MS=2000
+CONFIG_CLOCK_CONTROL_NRF2_NRFS_CLOCK_TIMEOUT_MS=1000
```

Comment:
These two defaults are set by the driver and can be removed from the test suite :)